rust/kernel/platdev.rs

```diff
@@ -107,3 +119,20 @@ impl Drop for Registration {
         }
     }
 }
+
+/// Trait for implementers of platform drivers.
+///
+/// Implement this trait whenever you create a platform driver.
```

Comment:
Super nit: here you're using the imperative mode "Implement this [...]" while in the functions below you use the indicative mode "Implementers should attempt [...]".

I think we should be consistent and use the same structure in both cases. Personally, I favor the indicative because we are providing information to the reader, not giving them orders. (Though admittedly opinions vary here.)

Reply:
Agreed ! And these are not super nits at all. Please do provide that info, so I can improve my "internal algorithm".